x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/session/IndexResolver.java

```diff
@@ -91,14 +92,49 @@ public void resolveAsMergedMapping(
         boolean supportsAggregateMetricDouble,
         boolean supportsDenseVector,
         ActionListener<IndexResolution> listener
+    ) {
+        ActionListener<Versioned<IndexResolution>> ignoreVersion = listener.delegateFailureAndWrap(
+            (l, versionedResolution) -> l.onResponse(versionedResolution.inner())
+        );
```

Comment:
Is this method without the version still being used? Should it?

Reply:
It's used! When resolving enriches and lookup join indices.

The clusters that the respective enriches and lookup joins will be executed on are dictated by the `FROM` pattern; so we already got the minimum transport version of all relevant clusters, and thus don't have to pass through the min transport version from the lookup/enrich resolution.